Merkle Rewards System/spec-history.md

```diff
@@ -24,4 +25,12 @@ This table shows which rewards specification was used for each rewards interval
 | 49-59 | [v3](./legacy/rewards-calculation-spec-v3.md) |
 | 60-75 | [v4](./legacy/rewards-calculation-spec-v4.md) |
 | 76-117 | [v5](./legacy/rewards-calculation-spec-v5.md) |
-| 118+ | [v6](./rewards-calculation-spec.md) |
+| 118-? | [v6](./legacy/rewards-calculation-spec-v6.md) |
+| ?-118 | [v7](./legacy/rewards-calculation-spec-v7.md) |
```

Comment:
not sure if there's an easy way to find the right numbers for prater. might have to check the rewards files, which i don't have readily available.

Reply:
I also don't have the prater rewards files but I think it's ok to ignore that as it's deprecated.